test/unit/models/Centres.test.js

```diff
@@ -200,4 +200,17 @@ describe('INTEGRATION CentreModel', () => {
   it('should throw exception when unable to get by name', () =>
     expect(Centres.getByName("invalid centre")).to.be.eventually.rejectedWith(ValidationError)
   );
+  it('should on destroy, destroy orphaned prebookings', () =>
+    Centres.destroy()
+      .then(() => expect(Prebooking.find()).to.eventually.have.length(0))
+  );
+  it('should on destroy, destroy orphaned movements', () =>
+    Centres.destroy()
+      .then(() => expect(Movement.find()).to.eventually.have.length(0))
+  );
+  it('should on destroy, destroy orphaned events', () =>
+    Centres.destroy()
+      .then(() => expect(Event.find()).to.eventually.have.length(0))
+  );
+
```

Comment:
(It may be over kill) I would suggest to create an Integration test to include this possible scenario:

Given centre 'xx' exists
And a Detainee with id 'yy' at Centre 'xx' exists
When centre 'xx' is destroyed
Then Detainee with id 'yy' is destroyed